### PR TITLE
Header

### PR DESCRIPTION
**Pull request by sardarMrashed**
things are expected to be add in this branch are

header
styling
HTML tree
hover and other effects

by me

### CODE REVIEW

**Comment by Ameer2000Mzori** [comment]
good job @sardarMrashed  i see you working very hard and very serious, keep going, tomorrow or inshallah we can start with javascript basics and i see that you learn fast please keep on that and also note a lot note note

File: src/index.html [hunk 1]
```diff
@@ -14,15 +14,14 @@
...
-        <ul class="containers"></ul>
+        <ul class="containers">
         <il class="containers-items">10</il>
         <il class="containers-items">20</il>
         <il class="containers-items">30</il>
         <il class="containers-items">40</il>
         <il class="containers-items">50</il>
         <il class="containers-items">60</il>
          </ul>
-
```
Comment (Ameer2000Mzori) [inline src/index.html]: instead of li use buttons and also use one ul

File: src/style.css [hunk 2]
```diff
@@ -6,7 +6,15 @@ body {
...
+}
+
+section {
   justify-content: center;
+   margin: auto;
+  height: auto;
+  background-color: rgb(145, 126, 126);
 }
 
 h1:hover {
```
Comment (Ameer2000Mzori) [inline src/style.css]: make hover effect for buttons not title, or you can leave the title hover effect but you need to add buttons hover effect

File: src/index.html [hunk 3]
```diff
@@ -11,6 +11,24 @@
...
+        <ul class="containers">
+        <il class="containers-items">10</il>
+        <il class="containers-items">20</il>
+        <il class="containers-items">30</il>
+        <il class="containers-items">40</il>
+        <il class="containers-items">50</il>
+        <il class="containers-items">60</il>
+         </ul>
+         <input type="number" id="number-input" number="number"0 required>
+         <input type="submit" value="favourite">
```
Comment (Ameer2000Mzori) [inline src/index.html]: use the normal button so we can use it the way we want

**Comment by Ameer2000Mzori** [comment]
so far you are doing great but please check the comments i have sent you so you build the project as required. @sardarMrashed  thank you

**Comment by Ameer2000Mzori** [comment]
I see that you did a great job with adding the hover effect and styling tomorrow at 2 pm my time we will for sure have a meeting

**Comment by sardarMrashed** [comment]
hello,I'm trying to work on javascript ?

**Comment by sardarMrashed** [comment]
Screenshot 2024-02-08 103615
why still my html it is look like this

**Comment by Ameer2000Mzori** [comment]
we just had a meeting together and @sardarMrashed  need to have solid understanding of Git and Github: 

=> Learn and master Git and GitHub 
=> Maybe the following links can help you learn those tools

- https://www.youtube.com/watch?v=RGOj5yH7evk

Best Regards
Ameer Ameen

File: src/index.html [hunk 4]
```diff
@@ -12,16 +12,16 @@
...
-         </nav>
+        <ul class="containers">
+        <il class="containers-items">10</il>
+        <il class="containers-items">20</il>
+        <il class="containers-items">30</il>
+        <il class="containers-items">40</il>
+        <il class="containers-items">50</il>
+        <il class="containers-items">60</il>
+         </ul>
          <input type="number" id="number-input" number="number" required>
```
Comment (Ameer2000Mzori) [inline src/index.html]: these changes are fixed now and returned to the original code

File: src/javascript.js [hunk 5]
```diff
@@ -1,5 +1,51 @@
...
+// es6 arrow function
+
+// 2- when any of the buttosn is clicked i want to see the background color
+// of favorite button is changed - NOT DONE
+
+// 3- when a value is intered in the input, then i want after clicking the favorite button
+// i want the title from Pokedex to change to the value was in side of the input - NOT DONE
+
+// i want your task (2, 3) that i give you under this line
+// -----------------------------------------------------
```
Comment (Ameer2000Mzori) [inline src/javascript.js]: these changes are fixed now and returned to the original code

**Comment by Ameer2000Mzori** [comment]
everything fine now try to do the following  =>

1- fix your terminal
2- and try your best to understand git and gihub

File: src/javascript.js [hunk 6]
```diff
@@ -1,32 +1,22 @@
-// how do we select elements in javascript ?
```
Comment (Ameer2000Mzori) [inline src/javascript.js]: good job you did task 2

**Comment by sardarMrashed** [comment]
Do you want to to clean js file ?

File: src/javascript.js [hunk 7]
```diff
@@ -4,5 +4,9 @@
 // selecte element :
 const numberinput = document.getElementById("number-input");
 const favouriteBtn = document.getElementById("favourite-Btn");
+// const header1 = document.getElementsByClassName("header")[0];
 
-// your function:
+favouriteBtn.addEventListener("click", () => {
+
+  console.log(`When you click no favourite it will count numbers of clicks ${numberInput}`);
+})
```
Comment (Ameer2000Mzori) [inline src/javascript.js]: @sardarMrashed good that you tried your best but.
you may misunderstood the task and its as follows =>

- when favouriteBtn is clicked i want to console log the value that is inside the number input
-imagine you put 1 in the number input then click on favouriteBtn ,  then in the console should appear 1 and so on.

**Review by Ameer2000Mzori — COMMENTED** [review]
please make changes as requested

File: src/javascript.js [hunk 8]
```diff
@@ -1,12 +1,13 @@
...
+// Select elements:
+const numberInput = document.getElementById("number-input");
 const favouriteBtn = document.getElementById("favourite-Btn");
-// const header1 = document.getElementsByClassName("header")[0];
 
 favouriteBtn.addEventListener("click", () => {
+ 
+  const inputValue = numberInput.value;
+  console.log(`The value inside the number input is: ${inputValue}`);
+});
```
Comment (Ameer2000Mzori) [inline src/javascript.js]: good job you did the task as requested

File: src/style.css [hunk 9]
```diff
@@ -86,6 +86,7 @@ h5 {
   text-align: center;
```
Comment (Ameer2000Mzori) [inline src/style.css:86]: i like the changes

**Comment by Ameer2000Mzori** [comment]
hi @sardarMrashed  sorry for being so late, you were looking for a course about git and GitHub.

i have the following link can help you ==>  https://www.udemy.com/course/git-and-github-bootcamp/

File: src/javascript.js [hunk 10]
```diff
@@ -284,10 +284,16 @@ console.log(arry[1])
...
 // what is the index of 2 ? : 
+//it is the index of (1)
 
 // ==> TASK TWO
 // => explain to me how forEach works 
-
+let arry1 = [1, 2, 3, 4 , 5, 6, 7, 8];
+arry1.forEach(( arr,element, index) => {
+    console.log("arr:", arr, "Index:", index, "element:", element);
+})
```
Comment (Ameer2000Mzori) [inline src/javascript.js]: ==> TASK ONE
=> Explain to me how indexOf works 

use indexOf to show me what is the index of 2 here 

let arry = [1, 2, 3];

what is the index of 2 ? :

Comment (Ameer2000Mzori) [inline src/javascript.js]: =>good job on task two, 


=>but task one you need to solve it, 
by using indexOf to see what is the index of 2 inside of the array

**Comment by Ameer2000Mzori** [comment]
==> TASK ONE @sardarMrashed  =>
=> Explain to me how indexOf works 

use indexOf to show me what is the index of 2 here 

let arry = [1, 2, 3];

what is the index of 2 ? :

**Comment by Ameer2000Mzori** [comment]
=> use indexOf method to show me what is the index of 2 here

File: src/javascript.js [hunk 11]
```diff
@@ -280,8 +280,7 @@ console.log(sorry we cant sent less then 1EU);
 // => explain to me how indexOf works 
 // he indexOf() method returns the first index (position) of a specified value.
 let arry = [1, 2, 3];
-console.log(arry[1])
-
+console.log(arry.indexOf(2));
 
 // what is the index of 2 ? : 
 //it is the index of (1)
```
Comment (Ameer2000Mzori) [inline src/javascript.js]: good job

**Comment by Ameer2000Mzori** [comment]
@sardarMrashed  thank you for collaborating with me in this amazing project

**Pull request merged**